app/src/components/Forms/ContributionReady/index.js

```diff
@@ -167,6 +171,21 @@ class ContributionReadyForm extends React.Component {
                   contributionId={values.id}
                   showOccupationLetter={visibleIf.showOccupationLetter}
                 />
+                {console.log({ matchId })}
+                {/* {isGovAdmin && matchId ? ( // ----KELLY - how call match id? */}
+                <div>
+                  <h2>Previous Donations</h2>
+                  <PreviousDonationsTable
+                    pastContributions={pastContributions}
```

Comment:
Looks like pastContributions is already called if there is a matchId in contribution duct. So no need to add it, pass it, or call it.

Reply:
ok.